projects/vdk-plugins/vdk-notebook/src/vdk/plugin/notebook/notebook_based_step.py

```diff
@@ -31,10 +30,21 @@ class NotebookStep(Step):
     A notebook step that will be executed when running a data job.
     """
 
-    def __init__(self, name, type, runner_func, file_path, job_dir, code, parent=None):
+    def __init__(
+        self,
+        name,
+        type,
+        runner_func,
+        file_path,
+        job_dir,
+        code,
+        module=None,
+        parent=None,
+    ):
```

Comment:
l'm looking for parameters documentation, it would be useful I think for code readability